common/telemetry.js

```diff
@@ -25,9 +25,17 @@ const networkInfoWriteClient = influx.getWriteApi(
   's' // precision
 )
 
+// Add new write client for batch metrics
```

Comment:
Let's not add new bucket and write client, rather let's just extend existing `publish` metric.